sections/Discussion.tex

```diff
@@ -0,0 +1,87 @@
+
+In the article in \autoref{fig:the_article}, we have highlighted the highest probable words within the three most occurring topics in the article.
+The article is about agriculture and how farmers are letting their cows out onto grass fields in September. 
+It also mentions a few different farms in the Northern part of Jutland and describes these in various ways.
+
+\begin{table}
+	\caption{Top 10 words for each model used in \autoref{tab:disc_color}.}
+	\label{tab:top_words_three_models}
+	\begin{tabular}{c|p{0.8\columnwidth}}
+		Topic & Top 10 words for \gls{lda} \\
+		\midrule
+		0 & sæby, a, direktør, frederikshavn, virksomheden, hans, medarbejdere, pedersen, firmaet, procent \\
+		1 & omradet, boliger, natur, naturen, ligger, du, vand, dyr, a, skov \\
+		2 & nielsen, arets, prisen, dansk, mors, jensen, løgstør, aars, vm, thy \\
+	\end{tabular}
+	\begin{tabular}{c|p{0.8\columnwidth}}
+		\midrule
+		Topic & Top 10 words for Author-Topic \\
+		\midrule
+		0 & du, procent, unge, børn, arige, hans, dansk, mig, thisted, mener\\
+		1 & sine, skriver, mig, børn, seneste, land, dansk, kommuner, andersen, formand \\
+		2 & set, glas, odense, vesthimmerland, leth, markedet, trump, ni, regionerne, prins\\
+	\end{tabular}
+	\begin{tabular}{c|p{0.8\columnwidth}}
+		\midrule
+		Topic & Top 10 words for Category-Topic \\
+		\midrule
+		0 & du, hans, børn, mig, thisted, procent, bedre, a, kr, maske \\
+		1 & klar, haft, fem, hjørring, ham, nyt, formand, min, aften, sagen\\
+		2 & du, min, gode, gamle, ad, henrik, eu, finde, sat, hobro\\
+	\end{tabular}
+\end{table}
+
+In \autoref{tab:top_words_three_models}, we can see that there are not many top words in the Author-Topic and Category-Topic model that appear in the article below.
+To compare these models, we take the top 200 words for the topic-word distributions within each model and mark them in the article below.
+We take 200 words since we want to see how intertwined the models are and because it is hard to find specific words for the Author and Category-Topic model since the topic coherence is low.
+Since the Author and Category models do not have a document-topic distribution we can not look at the specific document, but instead we have marked the words from the given category- and author-topic distribution within the document, to see what the difference in topics are.
+We are looking at the top 3 topics within each model for the specific document.
+\begin{table}[ht]
+	\centering
+	\caption{Color scheme for each model.}
+	\begin{tabular}{l|c}
+		Topic Model & Color \\
+		\midrule
+		\Acrlong{lda} & \thiscolor{Goldenrod} \vspace*{2mm} \\
+		Author-Topic Model & \thiscolor{Aquamarine} \vspace*{2mm} \\
+		Category-Topic Model & \thiscolor{LimeGreen} \vspace*{2mm} \\
+		Word appearing in all models & \thiscolor{Peach} \vspace*{2mm}  \\
+	\end{tabular}
+	\label{tab:disc_color}
+\end{table}
+\newline
```

Comment:
jeg syntes denne figur fylder for meget :/

Reply:
Vi kan godt lave noget negative spacing